### ISSUE TITLE
Writing on sys.stderr

### ISSUE THREAD

**Issue by zuphilip** [tmbdev/ocropy#132]
There is quite a bunch of statements `sys.stderr.write` where some information is written on standard error stream instead of normal standard output stream, cf. https://github.com/tmbdev/ocropy/search?utf8=%E2%9C%93&q=sys.stderr.write

**Comment by kba** [comment]
Is that a problem per se? Making logging consistent is a good thing but writing warnings/errors to STDERR can be useful as well.

**Comment by tmbdev** [comment]
Yes; if you want both stderr and stdout together, just run the command with 

    command ... args ... 2>&1

**Comment by zuphilip** [comment]
It is not a general problem, but I had the impression that maybe some of the print statements would be more consistent to use standard output instead, e.g. the information about the detected errors in [ocropus-econf](https://github.com/tmbdev/ocropy/blob/97ace5581b4c4deb95cfcd17b595b0ff1da36c25/ocropus-econf#L87-L97) is reported with these lines
```py
sys.stderr.write("errors    %8d\n"%errs)
sys.stderr.write("missing   %8d\n"%missing)
sys.stderr.write("total     %8d\n"%total)
sys.stderr.write("err       %8.3f %%\n"%(errs*100.0/total))
sys.stderr.write("errnomiss %8.3f %%\n"%((errs-missing)*100.0/total))

if args.confusion>0:
    for (a,b),v in counts.most_common(args.confusion):
        print "%d\t%s\t%s"%(v,a,b)

print errs*1.0/total
```
Thus, we have some `sys.stderr.write` and some normal `print` statements. Shouldn't we change some of these statements? (Please note that such changes in this file will currently interfere with https://github.com/tmbdev/ocropy/pull/131)

**Comment by stweil** [comment]
Using stderr for error messages is fine. Mixing stdout and stderr for normal program output results in several problems:

* There is no synchronization between stdout and stderr, so messages appear in an unexpected order (also if using `2>&1`). Typically stdout is buffered while stderr is unbuffered. For example `ocropus-err` outputs file results, then the summary followed by more file results while the summary is expected at the end.

* Users don't expect normal output on stderr. If they use something like `ocropus-err ... | less`, they will see output, but miss the summary.

Philipp, either apply PR #131 first (then I'll rebase #170), or apply #170 (then #131 can be rebased, either by you or by me).
